java/arcs/sdk/wasm/WasmSingletonImpl.kt

```diff
@@ -46,4 +46,6 @@ class WasmSingletonImpl<T : WasmEntity>(
             action(entity)
         }
     }
+
+    override fun getContent(): T? = fetch()
```

Comment:
We should be able to do this without adding new methods to the handle class - we don't want developers to be able to see both fetch() and getContent() and wonder what's the difference.

You can have generic behaviour thanks to the type of WasmHandleEvents, but you don't need to add new methods, instead you could switch between fetch() and fetchAll() inside the combine().